java/flight/flight-sql/src/main/java/org/apache/arrow/flight/sql/SqlInfoBuilder.java

```diff
@@ -502,6 +503,26 @@ public SqlInfoBuilder withSqlQuotedIdentifierCase(final SqlSupportedCaseSensitiv
     return withBitIntProvider(SqlInfo.SQL_QUOTED_IDENTIFIER_CASE_VALUE, value.getNumber());
   }
 
+  /**
+   * Sets a value for {@link SqlInfo#SQL_ALL_TABLES_ARE_SELECTABLE} in the builder.
+   *
+   * @param value the value for {@link SqlInfo#SQL_ALL_TABLES_ARE_SELECTABLE} to be set.
```

Comment:
nit: is this wording okay?

Reply:
I took `withSqlQuotedIdentifierCase` method javadoc as an example.